src/components/solverComponents/SolverTitle.tsx

```diff
@@ -0,0 +1,12 @@
+import { Heading, VStack } from "@chakra-ui/react";
+
+export const SolverTitle = (props: { title: string , text: string}) => (
+  <VStack
+    justifyContent="center"
+    alignItems="center"
+    height="60vh"
+  >
+    <Heading fontSize="3vw">{props.title}</Heading>
+    <Heading lineHeight='tall' fontSize='1vw'>{props.text}</Heading>
```

Comment:
This seems to small when using half a screen size to use the website. For the `Heading` component, there is a [`size` property](https://chakra-ui.com/docs/components/heading) which allows you to configure the responsive heading sizes of chakra ui.

![image](https://user-images.githubusercontent.com/60352933/192225668-a1e302b9-5cf9-4438-aa1e-3caf9eacf7d7.png)

Reply:
used the size property